src/main/resources/termsOfService.md

```diff
@@ -160,20 +162,20 @@ legitimate tasks or assigned duties. If you believe you are being
 granted access that you should not have, you must immediately notify the
 Broad Operations Center at infosec@broadinstitute.org.
 
-## Restricted Use of TCGA Controlled-Access Data
+## Restricted Use of Federal Controlled-Access Data
 
-To access TCGA controlled access data, you must first request to link
+To access federal controlled access data, you must first request to link
 your Terra account to the eRA Commons or NIH identity under which you
 are granted dbGaP authorized access. If you have dbGaP authorized
-access, Terra permits read access to the TCGA controlled-access data
-files.
+access, Terra permits read access to those the federal controlled-access
+data files.
 
 If the NIH or eRA commons identity authentication fails or the
 authenticated identity no longer appears in the NIH-provided whitelist,
-you can no longer access TCGA controlled access data files. It is your
-responsibility to delete any TCGA controlled-access data in your
+you can no longer access federal controlled access data files. It is your
+responsibility to delete any federal controlled-access data in your
 personal workspaces if you lose dbGaP authorized access. You must also
-refrain from distributing TCGA controlled access data to users unless
+refrain from distributing federal controlled access data to users unless
```

Comment:
Should be "controlled-access data", no?

Reply:
I see the text is pretty inconsistent with "controlled access" vs. "controlled-access". It seems that usually "controlled-access" is used to describe "data files", but "controlled access" for just data. that seems OK I guess.